stacker/lookups/__init__.py

```diff
@@ -38,10 +38,11 @@ def extract_lookups_from_string(value):
         raw = match.groups()[0]
         lookup_type = groupdict["type"]
         if not lookup_type:
-            raise ValueError("stacker only allows explicit lookup types "
+            raise ValueError("on value `%s`\n"
```

Comment:
I'd instead keep this error the way it is, but catch it at:

https://github.com/remind101/stacker/blob/release-1.0/stacker/variables.py#L149

Then we can raise an exception with the original message, but give both the variable name and value if necessary.